ajax/libs/jquery-typeahead/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "jquery-typeahead",
+  "filename": "jquery.typeahead.min.js",
+  "version": "2.10.4",
+  "description": "jQuery plugin that provides Typeahead (autocomplete) Search preview from Json object(s) via same domain Ajax request or cross domain Jsonp and offers data compression inside Local Storage. The plugin is built with a lot of options and callbacks to allow customization.",
+  "author": "Tom Bertrand",
+  "keywords": [
+    "search",
+    "typeahead",
+    "ajax",
+    "json",
+    "jsonp",
+    "input",
+    "autocomplete"
+  ],
+  "homepage": "http://www.runningcoder.org/jquerytypeahead/",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/running-coder/jquery-typeahead.git"
+  },
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/running-coder/jquery-typeahead.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
+        ]
+      },
+      {
+        "basePath": "src",
+        "files": [
+          "jquery.typeahead!(*.scss)"
```

Comment:
I think we use included mode when `src` as the bathPath.
Like `jquery.typeahead*.+(css|js|map)`
What do you think?
Thanks.

Reply:
It was my mistake, will update it, thx.